test/nodes/test_summarizer.py

```diff
@@ -1,125 +1,83 @@
-from haystack.nodes.other.document_merger import DocumentMerger
-
-
-pytestmark = pytest.mark.skip("Tests are too heavy for Github runners, skipping for now")
-
-
-DOCS = [
-    Document(
-        content="""PG&E stated it scheduled the blackouts in response to forecasts for high winds amid dry conditions. The aim is to reduce the risk of wildfires. Nearly 800 thousand customers were scheduled to be affected by the shutoffs which were expected to last through at least midday tomorrow."""
-    ),
-    Document(
-        content="""The tower is 324 metres (1,063 ft) tall, about the same height as an 81-storey building, and the tallest structure in Paris. Its base is square, measuring 125 metres (410 ft) on each side. During its construction, the Eiffel Tower surpassed the Washington Monument to become the tallest man-made structure in the world, a title it held for 41 years until the Chrysler Building in New York City was finished in 1930. It was the first structure to reach a height of 300 metres. Due to the addition of a broadcasting aerial at the top of the tower in 1957, it is now taller than the Chrysler Building by 5.2 metres (17 ft). Excluding transmitters, the Eiffel Tower is the second tallest free-standing structure in France after the Millau Viaduct."""
-    ),
-]
-
-EXPECTED_SUMMARIES = [
-    "California's largest electricity provider, PG&E, has shut down power supplies to thousands of customers.",
-    " The Eiffel Tower in Paris has officially opened its doors to the public.",
-]
-
-SPLIT_DOCS = [
-    Document(
-        content="""The tower is 324 metres (1,063 ft) tall, about the same height as an 81-storey building, and the tallest structure in Paris. Its base is square, measuring 125 metres (410 ft) on each side. During its construction, the Eiffel Tower surpassed the Washington Monument to become the tallest man-made structure in the world, a title it held for 41 years until the Chrysler Building in New York City was finished in 1930."""
-    ),
-    Document(
-        content="""It was the first structure to reach a height of 300 metres. Due to the addition of a broadcasting aerial at the top of the tower in 1957, it is now taller than the Chrysler Building by 5.2 metres (17 ft). Excluding transmitters, the Eiffel Tower is the second tallest free-standing structure in France after the Millau Viaduct."""
-    ),
-]
-
-# Documents order is very important to produce summary.
-# Different order of same documents produce different summary.
-EXPECTED_ONE_SUMMARIES = [
-    " The Eiffel Tower in Paris has officially opened its doors to the public.",
-    " The Eiffel Tower in Paris has become the tallest man-made structure in the world.",
-]
-
-
-@pytest.mark.integration
-@pytest.mark.summarizer
-def test_summarization(summarizer):
+from haystack.nodes import TransformersSummarizer
+
+
+DOCS = [Document(content=doc) for doc in ["First test doc", "Second test doc"]]
+SUMMARIES = ["First summary", "Second summary"]
```

Comment:
We refer to the items in this list later on as `expected_summary` when we iterate through the last so it would be also okay to call the list `EXPECTED_SUMMARIES` here.